video_mapper.go

```diff
@@ -420,7 +425,16 @@ func getByline(brightcoveVideo map[string]interface{}, publishReference string)
 	return byline
 }
 
-func get(key string, brightcoveVideo map[string]interface{}) (val string, err error) {
+func getFirstPublishedDate(video map[string]interface{}) (result string) {
+	result, err := get("published_at", video)
```

Comment:
Would it make sense to extract the field name into a variable? Although, I don't have strong views on this... Also, is it intentional to use the same field for publishDate and firstPublishDate?